samples/client/petstore/bash/Dockerfile

```diff
@@ -39,6 +39,7 @@ For convenience, you can export the following environment variables:\n\
 \n\
 $(tput setaf 3)PETSTORE_HOST$(tput sgr0) - server URL, e.g. https://example.com:8080\n\
 $(tput setaf 3)PETSTORE_API_KEY$(tput sgr0) - access token, e.g. "ASDASHJDG63456asdASSD"\n\
+$(tput setaf 3)PETSTORE_API_KEY$(tput sgr0) - access token, e.g. "ASDASHJDG63456asdASSD"\n\
```

Comment:
@ kenjones-cisco Why this line is added here? - it is an exact duplicate of the above line. Did you mean to add another env var in here or is it just a 'typo'?

Reply:
I didn't add it manually it happened when the code was generated.
I just reran the ./bin/ script for bash.

Reply:
Yes you're right, it seems the petstore example has changed since last time the bash examples were updated (in particular this change is probably the [cause](https://github.com/swagger-api/swagger-codegen/commit/82835db6701c41b7888c6e13d718c9936358b9f5#diff-45181bd22b428830b22bb957f1fe4772) in the repo, and accidentally with your PR the issue came up :-) I'll have a look at it and submit a separate PR... I